### ISSUE TITLE
The "Architecture" in "Image Details" isn't displayed correctly

### ISSUE THREAD

**Issue by ywk253100** [vmware/harbor#3023]
![tag_detail](https://user-images.githubusercontent.com/5835782/29154721-19288342-7dc9-11e7-8a08-71cc186921c2.png)

API response:
```
{
  "digest": "sha256:82009546cb8d393e1cc4bdd7a854bbda87dcf503cda9d8782b5467895bce5d7e",
  "name": "docker_1.9_signed",
  "architecture": "",
  "os": "",
  "docker_version": "",
  "author": "",
  "created": "0001-01-01T00:00:00Z",
  "signature": {
    "tag": "docker_1.9_signed",
    "hashes": {
      "sha256": "ggCVRsuNOT4cxL3XqFS72ofc9QPNqdh4K1RniVvOXX4="
    }
  },
  "scan_overview": {
    "image_digest": "sha256:82009546cb8d393e1cc4bdd7a854bbda87dcf503cda9d8782b5467895bce5d7e",
    "scan_status": "error",
    "job_id": 3,
    "severity": 0,
    "details_key": "",
    "creation_time": "2017-08-09T10:58:32Z",
    "update_time": "2017-08-10T00:00:08Z"
  }
}
```

**Comment by steven-zou** [comment]
@ywk253100 

>"architecture": "",
  "os": "",
  "docker_version": "",
  "author": "",

Seems API does not return the data.

**Comment by ywk253100** [comment]
Yes, but if the API returns null, I think UI should display null either rather than the date.

**Comment by steven-zou** [comment]
As issue is fixed in harbro-ui@0.4.46, close this issue
